### PR TITLE
Troubleshooting section added for those that lack rbenv in their path

### PR DESCRIPTION
**Pull request by Bastes**
Following [issue #9](https://github.com/oh-my-fish/plugin-rbenv/issues/9), I thought people encountering the same problem would benefit from a troubleshooting section explaining the issue and how to fix it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,26 @@ If you have a custom rbenv root, you can add the following to your `~/.config/fi
...
+If you did install rbenv and still receive the message `Please install 'rbenv'
+first!`, be sure to add your rbenv's command directory to your PATH in your
+`~/.config/fish/config.fish` before oh-my-fish's init file is sourced:
+
+```
+# ~/.config/fish/config.fish
+...
+
+# assuming your rbenv command is located under ~/.rbenv/bin
+set -g PATH $HOME/.rbenv/bin $PATH
```
Comment (bpinto) [inline README.md]: No need for -g

**Comment by bpinto** [comment]
Shouldn't we do this automatically inside this if block?

```
  if not set -q RBENV_ROOT
    set -x RBENV_ROOT "$HOME/.rbenv"
  end
```

File: README.md (hunk 1 above)
Comment (Bastes) [inline README.md]: Thanks, fixing it right away.

**Comment by Bastes** [comment]
@bpinto I'm not sure what you meant there ^^° but if you mean replace the "available rbenv" check and instead put something that looks up the $RBENV_ROOT path for the rbenv executable, I'm all for it. I'll try and give you a new PR when I'm done.

**Comment by bpinto** [comment]
Ping me once you have a PR, I'll be on a computer in the next few days so I'll become more helpful!! Replying on a mobile is a terrible experience.

**Comment by Bastes** [comment]
@bpinto terrible indeed ^^ stay tuned for the PR

**Comment by Bastes** [comment]
@bpinto There ; I added what we discussed (unless I've missed the point ^^°) and removed the now redundant troubleshooting section in the README ; it fixes the problem on my config at least (provided of course I added the $RBENV_ROOT variable in my fish.config) but it feels a bit patchy. Feel free to comment ^^

**Comment by alessandro-fazzi** [comment]
In order to avoid the arrow anti-pattern I'd suggest to do an easier to read (really not tested!)

``` fish
if set -q RBENV_ROOT and not contains "$RBENV_ROOT/bin" $PATH
  set PATH $RBENV_ROOT/bin $PATH
end

if not available rbenv
  echo "Please install 'rbenv' first, or set \$RBENV_ROOT!"; return 1
end
```

also if less sophisticated. What do you think?

Thanks for the PR.

Cheers :)

**Comment by Bastes** [comment]
@pioneerskies That might just work ^^ I'll test and send a more sophisticated PR when it passes. Thanks.

**Comment by bpinto** [comment]
That's indeed what I was thinking!

You could also use autoload function as it prevents duplicated paths too. Not sure what is better...

**Comment by Bastes** [comment]
@pioneerskies @bpinto Here goes, tested on my config ^^

@bpinto I'm not sure what to make of the autoload command, I found it in OMF's internals but I'm not quite sure what it does and how to use it (I tried to read the source but I guess I'm a bit too green to shell scripting to really get it ^^°).

File: rbenv.fish [hunk 2]
```diff
@@ -2,8 +2,12 @@
 #   rbenv plugin for oh my fish
 
 function init --on-event init_rbenv
+  if not available rbenv; and set -q RBENV_ROOT; and not contains "$RBENV_ROOT/bin" $PATH
```
Comment (bpinto) [inline rbenv.fish]: Unnecessary check for rbenv command?

**Comment by bpinto** [comment]
Could you squash your commits?

**Comment by bpinto** [comment]
I could walk you through the code next week if you are interested on understand that function.

File: rbenv.fish (hunk 2 above)
Comment (Bastes) [inline rbenv.fish]: It feels a little redudant, and it works without, so I'm not adamant about keeping it. It just felt a little redundant to add $RBENV_ROOT to the $PATH when the rbenv command is already available.

**Comment by Bastes** [comment]
@bpinto Squashing right away ^^ and I'd certainly be interested to know more about autoload anytime.

**Comment by Bastes** [comment]
@bpinto There it is, all squashy as it should.

**Comment by bpinto** [comment]
Super! Thanks for the great work done here!

**Comment by Bastes** [comment]
Thank you all for the guidance ^^

**Pull request merged**